Options.py

```diff
@@ -1,13 +1,15 @@
 from __future__ import annotations
+
 import abc
 import logging
-from copy import deepcopy
 import math
 import numbers
-import typing
 import random
+import typing
+from copy import deepcopy
+
+from schema import And, Optional, Or, Schema
 
-from schema import Schema, And, Or, Optional
 from Utils import get_fuzzy_results
```

Comment:
This looks odd.

I think most common is either no blank line or a single blank line between all `import`s and all `from ... import`.
Imports are typically sorted alphabetically.

Reply:
The way my IDE is setup, all builtin imports are grouped together, then all pip package imports, then all local project imports.

Ultimately, it's just personal preference and if need be, I can easily revert that tweak.

Reply:
PEP8 only requires you to sort each individual group. Groups are separated by a blank line. The 3-way split looks verbose to me.

It's generally OK here, but I'm not sure if reformatting all files you touch (assuming that's automatic) is a good idea.